lib/components/base/Badge.vue

```diff
@@ -32,8 +86,8 @@ import {
   BoxIcon,
   ListIcon,
   EyeOffIcon,
-  FileTextIcon,
-  XIcon,
+  // FileTextIcon,
+  // XIcon,
```

Comment:
Not sure why these were commented. The above should be changed so `DraftIcon` is `FileTextIcon`, etc